elide-graphql/src/test/java/com/yahoo/elide/graphql/PersistentResourceFetcherTest.java

```diff
@@ -159,13 +162,13 @@ protected void assertQueryEquals(String graphQLRequest, String expectedResponse,
             requestScope.saveOrCreateObjects();
         }
         requestScope.getTransaction().commit(requestScope);
-        Assert.assertEquals(result.getErrors().size(), 0, "Errors [" + errorsToString(result.getErrors()) + "]:");
+        assertEquals(result.getErrors().size(), 0, "Errors [" + errorsToString(result.getErrors()) + "]:");
```

Comment:
Whenever you switch an assert from testNG to Jupiter, you need to make sure the "expected" and "actual" variables are swapped.  Otherwise the error messages will be wrong when the tests fail.